scripts/e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+git_ref=$(git rev-parse --short HEAD)
+tower_openshift_project=tower-e2e-"$git_ref"
+
+echo "tower_openshift_project: '"$tower_openshift_project"'" >> e2e_test_config.yml
+echo "tower_openshift_password: '"$TOWER_OPENSHIFT_PASSWORD"'" >> e2e_test_config.yml
+echo "tower_openshift_username: '"$TOWER_OPENSHIFT_USERNAME"'" >> e2e_test_config.yml
+echo "tower_openshift_master_url: '"$OPENSHIFT_MASTER"'" >> e2e_test_config.yml
+
+echo "Installing tower on openshift: ${OPENSHIFT_MASTER}"
+ansible-playbook -i ./inventories/hosts \
+playbooks/install_tower.yml \
+--extra-vars "@e2e_test_config.yml"
+install_exit_code=$?
+echo "Install playbook ansible exit code: $install_exit_code"
+
+#Pass environment variables to config file
+echo "e2e_tower_license: '"$TOWER_LICENSE"'" >> e2e_test_config.yml
+echo "e2e_tower_password: '"$TOWER_PASSWORD"'" >> e2e_test_config.yml
+echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
+
+echo "Bootstrapping tower in openshift project: $tower_openshift_project"
+ansible-playbook -i ./inventories/hosts \
+playbooks/bootstrap.yml \
+--extra-vars "@e2e_test_config.yml"
+bootstrap_exit_code=$?
+echo "Boostrap playbook ansible exit code: $bootstrap_exit_code"
+
+
+oc delete project $tower_openshift_project
+exit_code=`expr $install_exit_code + $bootstrap_exit_code`
```

Comment:
What I'm doing here is adding the two exit codes for each of the playbooks. Reason being is that a failure in the playbook doesn't fail the script. If the both run successfully the exit code with be 0. If there is any failure the total of the two exit codes is provided (the bit that I think is somewhat hacky - I've worked around this by providing the actual exit code right after the playbook finishes. @pmccarthy

Reply:
Setting `set -e` or `set -o errexit` would cause the script to fail on the first error, currently it's just going to carry on trying to do things needlessly after it's already failed. While i suppose that gets round the issue of deleting the ns at the end, I'm not sure it's the best way to go about it.